scripts/globals/mobskills/Slaverous_Gale.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------
+--  Slaverous Gale
+--
+--  Description: Deals earth damage that inflicts Plague and Slow effects on targets in front of the caster
+--  Type: Magical (Earth)
+--
+--
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+---------------------------------------------
+function onMobSkillCheck(target,mob,skill)
+    return 0;
+end;
+
+function onMobWeaponSkill(target,mob,skill)
+	local typeEffectOne = EFFECT_PLAGUE;
+	local typeEffectTwo = EFFECT_SLOW;
+	MobStatusEffectMove(mob,target,typeEffectOne,1,3,60);
+	MobStatusEffectMove(mob,target,typeEffectTwo,1,3,60);
+	local dmgmod = 1; 
+    local info = MobMagicalMove(mob, target, skill, mob:getWeaponDmg() * math.random(4,6),ELE_EARTH,dmgmod,TP_NO_EFFECT);
```

Comment:
I just realized most of the indents I pointed out, are because you have hard tabs in yoru files. Configure your editor to use 4 spaces and unix line ends. If you are using one with [editor config support, install the plugin for that](http://editorconfig.org/#download) and it'll handle nearly everything for you. Notepad++, vim, visual studio, and pretty much anything I can think of have support for this.